src/command.c

```diff
@@ -323,21 +323,22 @@ static int dv_convert_conv_v0(struct device *dev, struct dmp_cmb *cmb,
 	}
 	if (copy_from_user(cmd, user_cmd, size)) {
 		kfree(cmd);
-		pr_warn(DRM_DEV_NAME ": copy_from_user() failed for %zu bytes\n", size);
+		pr_warn(DRM_DEV_NAME ": copy_from_user() failed for %zu bytes\n"
+				, size);
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket
and always place `,` after previous statement without spaces